src/c-writer.cc

```diff
@@ -431,8 +431,8 @@ class CWriter {
   void Spill(const TypeVector&);
   void Unspill(const TypeVector&);
 
-  template <typename Vars, typename TypeOf, typename ToDo>
-  void WriteVarsByType(const Vars&, const TypeOf&, const ToDo&);
+  template <typename Vars, typename TypeOfFunc, typename ToDo>
+  void WriteVarsByType(const Vars&, const TypeOfFunc&, const ToDo&);
```

Comment:
I don't think there is any need to replace the CamelCase versions of this string, just the argument name down on line 2960, right?